src/main.rs

```diff
@@ -210,7 +210,7 @@ fn print_source_line(
 
     let width = format!("{}", largest_line_number).len();
 
-    print!(" {:<width$} | ", line_number);
+    print!(" {:<width$} | ", line_number + 1);
```

Comment:
The correction should be done at the call site, or the argument renamed to line_index

Reply:
This would change the `largest_line_number` calculation also

Reply:
I renamed the argument to line_index since it is used like this on the call site.

Reply:
This doesn't fix what we suggest above. Since you're changing the meaning of the value, you need to start the counting with the initial and fix the calculation of `largest_line_number`